### DISCUSSION TITLE
Noob Question - Does Fort Firewall totally replaces Windows Firewall?

### DISCUSSION THREAD

**Discussion by JFMugen** [tnodir/fort#314] [Q&A] [answered]
Question in the title. Sorry if it is an obvious question. 

For example will Windows still bring me that 'Allow' firewall windows etc? Or FF simply disables Windows's Firewall and works alone?

**Comment by tnodir** [comment, marked as answer]
Fort Firewall (FFW) does not replace Windows Firewall (WF).

Usually WF does not show that 'Allow' dialog after FFW.

I do not recommend disabling the WF.

**Reply by JFMugen** [reply]
Thank you for the answer!

**Comment by kstrikis** [comment]
I found Fort because WF with Malwarebyte's WFC has some limitations. I will keep WF enabled, and will start using FFw.

I will continue using WFC as well. With both WFC and FFw, WFP appears to be the 1st stage filter--WFC's notification pops up on a new unknown program connection, and FFw does not. Then, after allowing that program with WFC, FFw will alert about the program.

So I will use WF/WFC in a permissive way, so that FFw can pick up the slack with its more powerful functions. I'm new to FFw so would like to hear of any concerns/quirks when doing this. So far looks good! I hope this order of filtering will be consistent.

**Reply by tnodir** [reply]
> WFC's notification pops up on a new unknown program connection, and FFw does not.

Which settings do you use in WFC? I'd like to reproduce this behavior in VM.

WFC's notification and FFw's alert popups appear at same time for me.

I'm wondering why FFw shows popup only after WFC's decision for you, because WFC can not pause connections.
